index.yaml

```diff
@@ -39,17 +48,36 @@ entries:
     version: v0.1.9
   - apiVersion: v1
     appVersion: v1
-    created: "2021-04-08T10:53:23.517675303Z"
+    created: "2021-07-13T14:23:53.475385396Z"
     description: A helm chart for armada API component
     digest: 09b01cd14bd92e3e75a9fb7356677464a8ad02025deee81cacb5482185d87a7c
     name: armada
     urls:
     - armada/armada-0.0.1.tgz
     version: 0.0.1
+  armada-binoculars:
+  - apiVersion: v1
+    appVersion: v0.1.42
+    created: "2021-07-13T14:23:53.483558837Z"
+    description: A helm chart for Armada Binoculars component
+    digest: a765fcae2e28ec8473a6cd70bbe1654f5d72fe646a14dadf2e3e1f8d09574b3f
+    name: armada-binoculars
+    urls:
+    - binoculars/armada-binoculars-v0.1.42.tgz
```

Comment:
where theese URLs should point? i have trouble to find these raw files on GH 🤔

Reply:
This PR is automatically generated by the armada release pipeline. @stackedsax will know more.

Reply:
Then there seems to be a bug because this PR does not include the `armada-binoculars` chart - @ljubon could you please check the current armada workflow?

Reply:
thanks to @robertdavidsmith we have a fix for `binoculars` - https://github.com/G-Research/armada/pull/627 🎉 
I'll publish manually missing charts for armada, and with next release, we should verify that all charts are in place 💪